src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets

```diff
@@ -97,8 +100,25 @@
              Condition="'@(_ReferenceFileNamesToRemove)' != '' or '@(_ReferenceCopyLocalPathsFileNamesToRemove)' != ''"/>
   </Target>
 
+  <Target Name="ValidatePackageVersions">
+    <ValidateProjectDependencyVersions Condition="Exists('$(ProjectJson)')"
+                                       ProjectJson="$(ProjectJson)"
+                                       ProhibitFloatingDependencies="$(ProhibitFloatingDependencies)"
+                                       ValidationPatterns="@(ValidationPattern)" />
+  </Target>
+
+  <Target Name="UpdateDependencies">
```

Comment:
I would prefer we simply have one update all target in corefx that calls this target with a list of ProjectJson files. Similar to https://github.com/dotnet/corefx/blob/master/build.proj#L26.